doc/Changelog.md

```diff
@@ -1,29 +1,36 @@
 Changelog {#Changelog}
 =========
 
-# master {#master}
+# Release 0.7 (09-12-2016)
+* [#353](https://github.com/BlueBrain/Livre/pull/353):
+  handle files with right extension and no schema
```

Comment:
Would you know without context what this means? 

* Allow specification of raw volumes without schema